src/main/resources/config/discord/components.json

```diff
@@ -589,4 +589,4 @@
       "style": 4
     }
   }
-}
```

Comment:
This is something I've configured in my project, should probably do the same for yours (if you care): https://stackoverflow.com/questions/10418975/how-to-change-line-ending-settings

Reply:
Ah, I see. I've configured it now.